### ISSUE TITLE
uMatrix not usable

### ISSUE THREAD

**Issue by lb7jf47wpqcn** [gorhill/uMatrix#873]
Hello!

First I want to say how much I love this extension. I have been using it for a month now but I really enjoy the control over every website you visit. Great work!

So now with the issue. I use a few profiles in Firefox, one for Work, one Personal... A few days ago I opened my Work profile and found that uMatrix isn't working at all. The only thing I see is a white container and the options on top. When I tried opening the settings for uMatrix it wouldn't let me so I had to open them from the extensions menu in Firefox. Anyway I restarted multiple times, reinstalled the extension but the issue is still the same. I noticed it appeared after the Privacy tab was moved to the Settings tab. The interesting part was that in my Personal profile uMatrix was working fine but apparently not for long. The next day I open my other profiles to see that uMatrix isn't working there too. When browsing websites the extension works just fine blocking everything but I can't enable temporarily rules for websites so browsing is impossible while the extensions is enabled.

I am uploading a screenshot of my display.
https://imgur.com/a/CF8Ae

Firefox 57.0.1

**Comment by gorhill** [comment]
You are not providing uMatrix version. Read carefully CONTRIBUTING please.

Unable to reproduce. Investigate more on your side, browser console, etc. Without actionable information, I will have to close this issue.

**Comment by lb7jf47wpqcn** [comment]
Hello!

I am not using any custom browser settings, I am using Firefox 57.0.1 on Arch Linux. I tried with a fresh browser profile but the issue was still there. My uMatrix version is 1.1.14. Interesting I noticed that my other profile has updated uMatrix to version 1.1.16 and the issue has disappeared so I will wait for the update on my other profiles.

Thank you for the support and sorry for the inconvenience.
